spread.yaml

```diff
@@ -66,6 +84,33 @@ prepare: |
     # this indicates that the server got reused, nothing to setup
     [ "$REUSE_PROJECT" != 1 ] || exit 0
 
+    # Disable burst limit so resetting the state quickly doesn't create problems.
+    mkdir -p /etc/systemd/system/snapd.service.d
+    cat <<EOF > /etc/systemd/system/snapd.service.d/local.conf
+    [Unit]
+    StartLimitInterval=0
+    [Service]
+    Environment=SNAPD_DEBUG_HTTP=7 SNAP_REEXEC=0
+    EOF
+
+    # adhoc backend and ubuntu-core system, nothing to setup
```

Comment:
I suggest dropping this comment. It's not very useful, and actually wrong. We are setting up stuff below.